kaolin/datasets/shapenet.py

```diff
@@ -193,6 +160,85 @@ def __getitem__(self, index):
         return {'data': data, 'attributes': attributes}
 
 
+class ShapeNet(KaolinDataset):
+    r"""ShapeNet Dataset class for meshes.
+    Args:
+        root (str): path to ShapeNet root directory
+        categories (list): List of categories to load from ShapeNet. This list may
+                           contain synset ids, class label names (for ShapeNetCore classes),
+                           or a combination of both.
+        train (bool): is using training split (default: True)
+        split (float): proportion of the dataset to split to the training datase
+    Returns:
+        .. code-block::
+        dict: {
+            attributes: {name: str, path: str, synset: str, label: str},
+            data: {vertices: torch.Tensor, faces: torch.Tensor}
+        }
+    Example:
+        >>> meshes = ShapeNet(root='../data/ShapeNet/')
+        >>> obj = meshes[0]
+        >>> obj['data'].vertices.shape
+        torch.Size([2133, 3])
+        >>> obj['data'].faces.shape
+        torch.Size([1910, 3])
+    """
+
+    def initialize(self, root: str, categories: list, train: bool = True, split: float = .7):
+        """Initialize the dataset
+           Args:
+               root (str): path to ShapeNet root directory
+               categories (list): List of categories to load from ShapeNet. This list may
+                                  contain synset ids, class label names (for ShapeNetCore classes),
+                                  or a combination of both.
+               train (bool): is using training split (default: True)
+               split (float): proportion of the dataset to split to the training dataset"""
```

Comment:
Similar comments as above for the docstrings.

Reply:
Fixed